getting-started/vs-integration/new-project-wizard.md

```diff
@@ -90,6 +90,7 @@ The **Create New Project** wizard provides the following project templates:
 * **Dashboard**&mdash;A basic dashboard layout that uses the TileLayout component for customizable dashboards, and showcases several ways individual blocks (tiles) can fetch data.
 * **Admin**&mdash;A small app that shows a dashboard for admin. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
 
+> The **Admin**, **Dashboard**, and **CRUD**, **Form**, **Chart** templates are now deprecated and are planned to be removed with Q1 2025 release. The source code for these projects will be moved to the [blazor-ui repo](https://github.com/telerik/blazor-ui) for continued access.
```

Comment:
```suggestion
> The **Admin**, **Dashboard**, **CRUD**, **Form**, and **Chart** templates are now deprecated and are planned to be removed with the Q1 2025 release. The source code for these projects will be moved to the [blazor-ui repo](https://github.com/telerik/blazor-ui) for continued access.
```

Reply:
"now" is not necessary. It makes sense only for people who will read this in the very near future.